docs/sql-pyspark-pandas-with-arrow.md

```diff
@@ -217,7 +217,7 @@ For detailed usage, please see [`pyspark.sql.functions.pandas_udf`](api/python/p
 
 ### Supported SQL Types
 
-Currently, all Spark SQL data types are supported by Arrow-based conversion except `MapType`,
+Currently, all Spark SQL data types are supported by Arrow-based conversion except
```

Comment:
MapType still cannot be supported for pandas udfs because Maps have only recently been added to pyarrow and will only be available in pyarrow 1.0, see ARROW-6904. Could you revert this?

Reply:
Ah, I see. Thanks for the check.

Reply:
Done. Reverted last 2 commits